sympy/algebras/quaternion.py

```diff
@@ -482,6 +509,18 @@ def integrate(self, *args):
     def rotate_point(pin, r):
         """Returns the coordinates of the point pin(a 3 tuple) after rotation.
 
+        Parameters
+        ==========
+        pin : tuple
+            A 3-element tuple of coordinates of a point. This point will be
+            the axis of rotation.
+        r
+            Angle to be rotated.
+        Returns
+        =======
+        tuple
+            The coordinates of the quaternion after rotation.
+
```

Comment:
This section is not correctly rendered now.
You may have to add a spacing before `Returns`
<img src="https://user-images.githubusercontent.com/34944973/52391648-aa056800-2ae1-11e9-8b7c-dc2a8e22d2e3.png" width="500px">